rasa_core/training/generator.py

```diff
@@ -315,25 +237,29 @@ def _process_step(self, step, incoming_trackers):
         # need to copy the tracker as multiple story steps
         # might start with the same checkpoint and all of them
         # will use the same set of incoming trackers
-        trackers = [tracker.create_copy() for tracker in
+        trackers = [deepcopy(tracker) for tracker in
```

Comment:
please avoid importing functions from somewhere, better call them on the package (e.g. `copy.deepcopy(...)`) as this makes clear where that function is coming from.